File: 03Greedy/3-1_거스름돈.py
#-3-1 거스름돈
coin_type=[500,100,50,10]
n=int(input())
cnt=0
for coin in coin_type:
    cnt+=n//coin #해당 화폐로 거술러 줄 수 있는 동전의 개수
    n%=coin
print(cnt)

File: 03Greedy/3 - 1 거스름돈.cpp
#include <iostream>

using namespace std;

int main(void) {
	int n;
	int coins[4] = { 500, 100, 50, 10 };

	cin >> n;

	int cnt = 0;
	for (int& coin : coins) {
		cnt += (n / coin);
		n %= coin;
	}

	cout << cnt;
	return 0;
}
